packages/celotool/src/e2e-tests/transfer_tests.ts

```diff
@@ -168,7 +168,14 @@ describe('Transfer tests', function(this: any) {
   const gethConfig: GethTestConfig = {
     migrateTo: 18,
     instances: [
-      { name: 'validator', validating: true, syncmode: 'full', port: 30303, rpcport: 8545 },
+      {
+        name: 'validator',
+        validating: true,
+        syncmode: 'full',
+        port: 30303,
+        rpcport: 8545,
+        validator: validatorAddress,
```

Comment:
It doesn't look like we're testing proposer transaction fees can be sent to a different address, can we set `txFeeRecipient` here as well?